examples/async-counter/client.js

```diff
@@ -0,0 +1,44 @@
+const choo = require('../../')
+const html = require('../../html')
+const plur = require('plur')
+
+const app = choo()
+app.model({
+  state: {
+    counter: 0
+  },
+  reducers: {
+    increment: (data, state) => ({ counter: state.counter + 1 }),
+    decrement: (data, state) => ({ counter: state.counter - 1 })
+  },
+  effects: {
+    incrementAsync: function (data, state, send, done) {
+      setTimeout(() => send('increment', done), 1000)
```

Comment:
This is totally valid too; `data` is optional so is set to `null` if not passed explicitely.

[Ref in `barracks` codebase](https://github.com/yoshuawuyts/barracks/blob/master/index.js#L108-L111)